_data/toc.yaml

```diff
@@ -1667,7 +1667,7 @@ manuals:
         - path: /ee/ucp/admin/configure/configure-rbac-kube/
           title: Configure native Kubernetes role-based access control
         - path: /ee/ucp/admin/configure/enable-audit-logging/
-          title: Create UCP audit logs
+          title: Enable audit logging on UCP
```

Comment:
Should this now read "Audit logging on UCP", to be in line with the change you made to the heading?